### PR TITLE
Create mail.icehead.json

### PR DESCRIPTION
**Pull request by Tanjimfuad**
### Requirements
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. *This requirement is optional if you are registering a domain for emails.*
- [x] The website is reachable.
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.
   - *You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.*

### Link to Website

icehead.is-a.dev

### CODE REVIEW

File: domains/mail.icehead.json
Comment (WilliamDavidHarrison) [inline domains/mail.icehead.json:11]: You only need one record key not 2 so please combine the two.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/mail.icehead.json
Comment (Tanjimfuad) [reply domains/mail.icehead.json]: I think I fixed it?

**Review by wdhdev — DISMISSED** [review]

**Comment by Tanjimfuad** [comment]
And now?

**Comment by WilliamDavidHarrison** [comment]
> And now?

I cannot merge as I'm only a helper, please wait for a maintainer to merge the PR.

**Comment by Tanjimfuad** [comment]
Thanks

**Comment by Tanjimfuad** [comment]
But why

**Comment by WilliamDavidHarrison** [comment]
> But why

Helpers don't have those permissions.

**Review by MaskDuck — APPROVED** [review]
btw helpers can't merge prs because they don't have permissions to do that

i'd be very happy if helpers can merge prs

**Pull request merged**
